products/compute/terraform.yaml

```diff
@@ -12,6 +12,28 @@
 # limitations under the License.
 
 --- !ruby/object:Provider::Terraform::Config
+overrides:
+  - !ruby/object:Provider::Terraform::Override
+    name: 'HttpHealthCheck'
+    description: |
+      {{description}}
+
+      ~> **Note:** google_compute_http_health_check is a legacy health check.
+      The newer [google_compute_health_check](/docs/providers/google/r/compute_health_check.html)
+      should be preferred for all uses except
+      [Network Load Balancers](https://cloud.google.com/compute/docs/load-balancing/network/)
+      which still require the legacy version.
+  - !ruby/object:Provider::Terraform::Override
+    name: 'HttpsHealthCheck'
+    description: |
+      {{description}}
```

Comment:
If we use a different marker we can make a generic "traverse through all override properties" and process them without the need to call them out one by one like you did in abstrace_override.rb:35. Similar to how we did spread_api.

How about something like: <<[description]>>

Reply:
Not sure I follow your idea here.

I thought we had settled on `{{description}}` marker to be consistent with the other marker in api.yaml. For instance:
```yaml
base_url: projects/{{project}}/global/httpsHealthChecks
```

Reply:
Yes, but we're to process every property blindly, we would accidentally hit base_url and expand them incorrectly.